### DISCUSSION TITLE
can't protection desync kick ?

### DISCUSSION THREAD

**Discussion by iwanttodo1t** [YimMenu/YimMenu#609] [Q&A] [answered]
can't protection desync kick ?

**Comment by SkiddyToast** [comment, marked as answer]
It’s not desync kick that isn’t being blocked. It’s breakup kick, which is unblockable.

**Reply by iwanttodo1t** [reply]
Thank you for your reply

**Reply by maybegreat48** [reply]
Technically it can be blocked if you are the host and the person that tries to kick you joins after when you become host. You basically randomize m_peer_id_2 in send_net_info_to_lobby so the joinee does not have a proper peer id of you. Am planning on adding this to YimMenu once I figure out how to fix the chat

**Reply by SkiddyToast** [reply]
Wouldn’t that break other netMessages as well?

**Reply by maybegreat48** [reply]
Yes, it does break a few, including CMsgTextMessage. But they can be fixed by hooking them and rewriting the peer id
